specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,3090 @@
+        }
+      },
+      "description": "if showStats=true was specified in the request this field will contain information about the document payload."
+    },
+    "RequestStatistics": {
+      "type": "object",
+      "required": [
+        "documentsCount",
+        "validDocumentsCount",
+        "erroneousDocumentsCount",
+        "transactionsCount"
+      ],
+      "properties": {
+        "documentsCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of documents submitted in the request."
+        },
+        "validDocumentsCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of valid documents. This excludes empty, over-size limit or non-supported languages documents."
+        },
+        "erroneousDocumentsCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of invalid documents. This includes empty, over-size limit or non-supported languages documents."
+        },
+        "transactionsCount": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Number of transactions for the request."
+        }
+      },
+      "description": "if showStats=true was specified in the request this field will contain information about the request payload."
+    },
+    "MultiLanguageBatchInput": {
+      "type": "object",
+      "description": "Contains a set of input documents to be analyzed by the service.",
+      "required": [
+        "documents"
+      ],
+      "properties": {
+        "documents": {
+          "type": "array",
+          "description": "The set of documents to process as part of this batch.",
+          "items": {
+            "$ref": "#/definitions/MultiLanguageInput"
+          }
+        }
+      }
+    },
+    "MultiLanguageInput": {
+      "type": "object",
+      "description": "Contains an input document to be analyzed by the service.",
+      "required": [
+        "id",
+        "text"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "A unique, non-empty document identifier."
+        },
+        "text": {
+          "type": "string",
+          "description": "The input text to process."
+        },
+        "language": {
+          "type": "string",
+          "description": "(Optional) This is the 2 letter ISO 639-1 representation of a language. For example, use \"en\" for English; \"es\" for Spanish etc. If not set, use \"en\" for English as default. Use \"auto\" for automatic language detection."
+        },
+        "defaultLanguage": {
+          "type": "string",
+          "description": "(Optional) A 2 letter ISO 639-1 representation of a language to be used as a default value when the \"language\" property is set to \"auto\". The default value will be used as a fallback if a language cannot be automatically detected."
+        }
```

Comment:
still catching up on the rest api board meeting feedback, was it decided to move this to the task level or will we keep it under input?

Reply:
It will be in task level.

@abhahn Since we are not making this feature in time for Feb release, would you move LD related swagger changes to the March one?

Reply:
@kayousef do you mean the language detection tasks or just the auto detect / default language feature won't make it into the Feb release?

Reply:
@kristapratico Yes. Due to changes on our side in addition to the feedback we got from the board review, we will be pushing this to the next version (March).

Reply:
Sure, I think this was a change I originally forgot, but I will go ahead an move it to the next preview version Swagger that I'm working on instead.

Reply:
I've removed the changes for automatic language detection.